File: src/main/java/net/softbell/bsh/dto/request/NodeGroupPermissionDto.java
package net.softbell.bsh.dto.request;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

/**
 * @Author : Bell(bell@softbell.net)
 * @Description : 노드 그룹 권한 정보 DTO
 */
@Getter
@Setter
@ToString
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class NodeGroupPermissionDto
{
    private Long memberGid;
    private Integer permission;
}

File: src/main/kotlin/net/softbell/bsh/dto/request/NodeGroupPermissionDto.kt
package net.softbell.bsh.dto.request

/**
 * @author : Bell(bell@softbell.net)
 * @description : 노드 그룹 권한 정보 DTO
 */
data class NodeGroupPermissionDto (
        var memberGid: Long,
        var permission: Int
)